dev-lua/luafilesystem/files/luafilesystem-1.7.0.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/Makefile b/Makefile
+index e50d5a0..dfc1a8a 100644
+--- a/Makefile
++++ b/Makefile
+@@ -18,8 +18,8 @@ test: lib
+ 	LUA_CPATH=./src/?.so lua tests/test.lua
+ 
+ install:
+-	mkdir -p $(LUA_LIBDIR)
+-	cp src/lfs.so $(LUA_LIBDIR)
++	mkdir -p $(DESTDIR)$(LUA_LIBDIR)
++	cp src/lfs.so $(DESTDIR)$(LUA_LIBDIR)
+ 
+ clean:
+ 	rm -f src/lfs.so $(OBJS)
+diff --git a/config b/config
+index cfd4c6a..915513b 100644
+--- a/config
++++ b/config
+@@ -1,10 +1,11 @@
+ # Installation directories
+ 
+ # Default installation prefix
+-PREFIX=/usr/local
++PREFIX=/usr
+ 
++LUA_VERSION=$(shell readlink -e "${EROOT}"/usr/bin/lua | sed -ne 's:.*/usr/bin/lua\([\d.-]*\):\1:p')
```

Comment:
Not saying this is wrong, but it looks a bit scary and IMO error-prone. Is there a way to "manipulate" these values by exporting them to ENV, for example, would `emake PREFIX=/usr` work overwriting this value?

Reply:
that works for the prefix.  However, since I've had to add a way to determine the lua version, I just stuck that part in the patch as well.